mypy/plugins/default.py

```diff
@@ -72,6 +76,10 @@ def get_method_hook(self, fullname: str
             return ctypes.array_iter_callback
         elif fullname == 'pathlib.Path.open':
             return path_open_callback
+        elif fullname == '{}.register'.format(singledispatch.SINGLEDISPATCH_TYPE):
```

Comment:
Compute these names once and store them in Final globals, instead of recomputing each time

Reply:
This should be fixed in 98359cbf6a35a38e6080c93dfac0af5d0357ec94.